### DISCUSSION TITLE
MACOS 13 ventura  is  not  support  macos vm

### DISCUSSION THREAD

**Discussion by summer-father** [utmapp/UTM#4653] [UTM Help (macOS)] [answered]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
Replace this text with a clear and concise description of what the issue is.

**Configuration**  mac auto create mac vm （download--run-cannot run）
* UTM Version: 4.0.9（68）
* macOS Version: MACOS 13 ventura
* Mac Chip (M2): 

**Crash log**  
If the app crashed, you need a crash log. To get your crash log, open Console.app, go to `Crash Reports`, and find the latest entry for either UTM, QEMU, QEMUHelper, or qemu-\*. Right click and choose `Reveal in Finder`. Attach the report here.

**Debug log**  
For all issues related to running a VM, _including_ crashes, you should attach a debug log. (This is unavailable for macOS-on-macOS VMs. Attach an excerpt of your system log instead.)
To get the Debug log: open UTM, and open the settings for the VM you wish to launch. Near the top of the `QEMU` page is `Debug Log`. Turn it on and save the VM. After you experience the issue, open the VM settings again and select `Export Log...` and attach it here.

**Upload VM**  
If your issue is related to a specific VM, please upload the config.plist from your VM's .utm directory. To get this, right-click the VM in UTM and choose "Show in Finder". Then right-click the selected file in Finder and choose "Show Package Contents". The config.plist file is now visible. Right-click it and choose "Compress". Attach the resulting config.plist.zip file here.
You can upload the entire .utm if needed, but note this includes your VM's drive image and may contain personal data. Since Github has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by summer-father** [comment]
click the  button run icon ---it  is  show  message   “This virtual machine cannot be run on this machine.”

**Reply by Ahsan-R-Kazmi** [reply]
I was able to get it working by manually choosing the IPSW for the boot image and using the IPSW that UTM downloads for the latest macOS. The very first time it seems that UTM downloads the IPSW file and it is available at the location `~/Library/Containers/com.utmapp.UTM/Data/Library/Caches`. When the message `This virtual machine cannot be run on this machine` appears the first time you try to run the VM, do not turn off / quit UTM. Instead go to the location `~/Library/Containers/com.utmapp.UTM/Data/Library/Caches` and copy the downloaded IPSW somewhere else, such as `~/Downloads`. Then quit UTM and restart it. When it starts, right click the VM and select `Edit > Boot` and choose the IPSW file for the `IPSW Install Image` field. (You will notice that the field is empty, since UTM clears the cache after it is quit, and deletes the IPSW file at the cached location. This is why you need to copy it somewhere else.)

**Reply by davekingery** [reply]
Thank you, I just wanted to note that in my case, the com.utmapp.UTM folder doesn't exist but I found the same path with UTM instead. Also, just a note that to get to the Library folder you should use the "Go" menu on finder and hold down Option to show "Library" as an option:
> ~/Library/Containers/UTM/Data/Library/Caches

**Comment by Ahsan-R-Kazmi** [comment]
Currently, experiencing this as well. After downloading the latest macOS IPSW through UTM and trying to run the new VM, an error is displayed saying: `This virtual machine cannot be run on this machine.` This is also on an M1 Mac running macOS 13.0.

Seems this may be a duplicate of: https://github.com/utmapp/UTM/issues/4595.

**Comment by osy** [comment]
I pressed the wrong button. Meant to say it’s a duplicate of https://github.com/utmapp/UTM/issues/4595

**Comment by syntien** [comment]
I am successfully running macOS Ventura 13.0.1 guest on macOS Ventura 13.0 running UTM Version 4.0.9 (68) on Mac Studio M1 Max (not an M2 machine).

![image](https://user-images.githubusercontent.com/1848471/201429546-5f04b126-82f0-456a-8752-ea847b733225.png)

**Reply by Ahsan-R-Kazmi** [reply]
Did you create a new guest macOS `13.0.1` VM on a host running macOS `13.0`? Perhaps you upgraded the guest from `12.x` to `13.0.1`. It seems there is an issue when trying to create the VM initially.

**Reply by syntien** [reply]
Directly created a new Ventura 13.0.1 guest VM

**Comment by summer-father** [comment]
first:  chose first
[截屏2022-11-12 11.12.20.png]

second:
then。auto。create

2022年11月12日 07:18，syntien ***@***.***> 写道：



Directly created Ventura 13.0.1 guest VM

**Comment by summer-father** [comment]
[截屏2022-11-12 11.16.01.png]

2022年11月12日 07:18，syntien ***@***.***> 写道：



Directly created Ventura 13.0.1 guest VM

**Comment by summer-father** [comment]
Third: next
[截屏2022-11-12 11.19.42.png]

2022年11月12日 11:19，zhao chenzhou ***@***.***> 写道：

<截屏2022-11-12 11.16.01.png>

2022年11月12日 07:18，syntien ***@***.***> 写道：



Directly created Ventura 13.0.1 guest VM

**Comment by summer-father** [comment]
Util save
[截屏2022-11-12 11.21.24.png]

2022年11月12日 11:19，zhao chenzhou ***@***.***> 写道：

<截屏2022-11-12 11.16.01.png>

2022年11月12日 07:18，syntien ***@***.***> 写道：



Directly created Ventura 13.0.1 guest VM

**Comment by summer-father** [comment]
Then, it atuo。download。from your。site ten。it will。run。auto—(at this。time。it will。be。err。ocur)
[截屏2022-11-12 11.22.12.png]

2022年11月12日 11:22，zhao chenzhou ***@***.***> 写道：

Util save
<截屏2022-11-12 11.21.24.png>

2022年11月12日 11:19，zhao chenzhou ***@***.***> 写道：

<截屏2022-11-12 11.16.01.png>

2022年11月12日 07:18，syntien ***@***.***> 写道：



Directly created Ventura 13.0.1 guest VM

**Comment by summer-father** [comment]
it is。show。err。time。out
[截屏2022-11-12 12.50.33.png]

2022年11月12日 11:22，zhao chenzhou ***@***.***> 写道：

Util save
<截屏2022-11-12 11.21.24.png>

2022年11月12日 11:19，zhao chenzhou ***@***.***> 写道：

<截屏2022-11-12 11.16.01.png>

2022年11月12日 07:18，syntien ***@***.***> 写道：



Directly created Ventura 13.0.1 guest VM

**Comment by summer-father** [comment]
it download。OK。sometimes

[截屏2022-11-12 16.30.55.png]

2022年11月12日 12:51，zhao chenzhou ***@***.***> 写道：


it is。show。err。time。out
<截屏2022-11-12 12.50.33.png>

2022年11月12日 11:22，zhao chenzhou ***@***.***> 写道：

Util save
<截屏2022-11-12 11.21.24.png>

2022年11月12日 11:19，zhao chenzhou ***@***.***> 写道：

<截屏2022-11-12 11.16.01.png>

2022年11月12日 07:18，syntien ***@***.***> 写道：



Directly created Ventura 13.0.1 guest VM

**Comment by summer-father** [comment]
click the。run。booton ,show。it。like this
[截屏2022-11-12 16.32.31.png]

2022年11月12日 16:32，zhao chenzhou ***@***.***> 写道：

it download。OK。sometimes

[截屏2022-11-12 16.30.55.png]

2022年11月12日 12:51，zhao chenzhou ***@***.***> 写道：


it is。show。err。time。out
<截屏2022-11-12 12.50.33.png>

2022年11月12日 11:22，zhao chenzhou ***@***.***> 写道：

Util save
<截屏2022-11-12 11.21.24.png>

2022年11月12日 11:19，zhao chenzhou ***@***.***> 写道：

<截屏2022-11-12 11.16.01.png>

2022年11月12日 07:18，syntien ***@***.***> 写道：



Directly created Ventura 13.0.1 guest VM

**Comment by summer-father** [comment]
how。it run。import。with。download。ipsw。image

2022年11月13日 13:15，Ahsan Kazmi ***@***.***> 写道：



I was able to get it working by manually choosing the IPSW for the boot image and using the IPSW that UTM downloads for the latest macOS. The very first time it seems that UTM downloads the IPSW file and it is available at the location ~Library/Containers/com.utmapp.UTM/Data/Library/Caches. When the message This virtual machine cannot be run on this machine appears the first time you try to run the VM, do not turn of UTM. Instead go the location ~Library/Containers/com.utmapp.UTM/Data/Library/Caches and copy the downloaded IPSW somewhere else, such as ~/Downloads. Then quit UTM and the next time it starts right click the VM and select Edit > Boot and choose the IPSW file for the IPSW Install Image. (You will notice that the field is empty, since UTM clears the cache after it is quit.)

**Comment by summer-father** [comment]
it。can。run。with。manual。ipsw image,  how。it。use。 apple not。modify。like。iso。or dmg。image。?

2022年11月13日 17:55，zhao chenzhou ***@***.***> 写道：

how。it run。import。with。download。ipsw。image

2022年11月13日 13:15，Ahsan Kazmi ***@***.***> 写道：



I was able to get it working by manually choosing the IPSW for the boot image and using the IPSW that UTM downloads for the latest macOS. The very first time it seems that UTM downloads the IPSW file and it is available at the location ~Library/Containers/com.utmapp.UTM/Data/Library/Caches. When the message This virtual machine cannot be run on this machine appears the first time you try to run the VM, do not turn of UTM. Instead go the location ~Library/Containers/com.utmapp.UTM/Data/Library/Caches and copy the downloaded IPSW somewhere else, such as ~/Downloads. Then quit UTM and the next time it starts right click the VM and select Edit > Boot and choose the IPSW file for the IPSW Install Image. (You will notice that the field is empty, since UTM clears the cache after it is quit.)

**Comment by summer-father** [comment]
&&。rockey。arm 8.6 is。not。support。on。iso。install utm too

2022年11月14日 07:28，zhao chenzhou ***@***.***> 写道：


it。can。run。with。manual。ipsw image,  how。it。use。 apple not。modify。like。iso。or dmg。image。?

2022年11月13日 17:55，zhao chenzhou ***@***.***> 写道：

how。it run。import。with。download。ipsw。image

2022年11月13日 13:15，Ahsan Kazmi ***@***.***> 写道：



I was able to get it working by manually choosing the IPSW for the boot image and using the IPSW that UTM downloads for the latest macOS. The very first time it seems that UTM downloads the IPSW file and it is available at the location ~Library/Containers/com.utmapp.UTM/Data/Library/Caches. When the message This virtual machine cannot be run on this machine appears the first time you try to run the VM, do not turn of UTM. Instead go the location ~Library/Containers/com.utmapp.UTM/Data/Library/Caches and copy the downloaded IPSW somewhere else, such as ~/Downloads. Then quit UTM and the next time it starts right click the VM and select Edit > Boot and choose the IPSW file for the IPSW Install Image. (You will notice that the field is empty, since UTM clears the cache after it is quit.)

**Comment by tr0hvx2u** [comment, marked as answer]
I tried three times to create macOS VM, but everything ended with a black screen.

Maybe my solution will be helpful. You can manually download IPSW from https://ipsw.me/product/mac. Then create a new VM and choose the current IPSW.
Or you can copy IPSW from `~/Library/Containers/com.utmapp.UTM/Data/Library/Caches` before starting VM.

**Reply by summer-father** [reply]
yes,it。works,  but。it。not。very。easy。for a。beginer

**Comment by summer-father** [comment]
It planed to fit the.kubevirt ?  For. Kuienets run. Vm in. The. Mac or. hypervisor. On. Mac or. kvm qemu?

2022年11月20日 19:02，Sergei Mirasov ***@***.***> 写道：



I tried three times to create macOS VM, but everything ended with a black screen.

Maybe my solution will be helpful. You can manually download IPSW from https://ipsw.me/product/mac. Then create a new VM and choose the current IPSW.
Or you can copy IPSW from ~/Library/Containers/com.utmapp.UTM/Data/Library/Caches before starting VM.

**Comment by summer-father** [comment]
I think. It will be. Funny. && interesting

2022年11月21日 07:28，zhao chenzhou ***@***.***> 写道：

It planed to fit the.kubevirt ?  For. Kuienets run. Vm in. The. Mac or. hypervisor. On. Mac or. kvm qemu?

2022年11月20日 19:02，Sergei Mirasov ***@***.***> 写道：



I tried three times to create macOS VM, but everything ended with a black screen.

Maybe my solution will be helpful. You can manually download IPSW from https://ipsw.me/product/mac. Then create a new VM and choose the current IPSW.
Or you can copy IPSW from ~/Library/Containers/com.utmapp.UTM/Data/Library/Caches before starting VM.
